package.json

```diff
@@ -235,6 +235,8 @@
     "ismobilejs": "^0.4.0",
     "js-yaml": "^3.13.1",
     "jsonschema": "^1.1.1",
+    "launchdarkly-js-client-sdk": "^2.12.0",
```

Comment:
didn't really demo this- you can get access to the flags with this framework by subscribing to `ready` and `on` update events.